spec/logux_spec.rb

```diff
@@ -23,4 +23,27 @@
       expect(subject).not_to be_empty
     end
   end
+
+  describe '.undo' do
+    subject { described_class.undo(meta, reason) }
+
+    let(:meta) do
+      {
+        id: '1 1:uuid 0',
+        users: ['3'],
+        reasons: ['user/1/lastValue'],
+        nodeIds: ['2:uuid'],
+        channels: ['user/1']
+      }
+    end
+
+    let(:reason) { 'error' }
+
+    before { stub_request(:post, Logux.configuration.logux_host) }
+
+    it 'makes request' do
+      subject
+      expect(WebMock).to have_requested(:post, Logux.configuration.logux_host)
```

Comment:
[Here](https://github.com/wilddima/logux_rails/blob/master/lib/logux/test/helpers.rb#L25) is new implemented matcher to check request is made

Reply:
@dsalahutdinov fixed